packages/components/src/frame-provider/index.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * External dependencies
+ */
+import { CacheProvider } from '@emotion/core';
+import createCache from '@emotion/cache';
+import memoize from 'memize';
+
+const memoizedCreateCacheWithContainer = memoize( ( container ) => {
+	return createCache( { container } );
+} );
+
+export default function FrameProvider( { children, iframeDocument } ) {
+	if ( ! iframeDocument ) {
```

Comment:
It looks like this component should never be rendered if there's no document.

Reply:
It's already the case, I guess this is just extra security.